Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -2018,3 +2040,76 @@ referralPopUpConfig state =
         }
       }
   in popUpConfig'
+
+type TipConfig = {
+  customerTipArray :: Array String,
+  customerTipArrayWithValues :: Array Int
+} 
+
+type TipVehicleConfig = {
+  sedan :: TipConfig,
+  suv :: TipConfig,
+  hatchback :: TipConfig,
+  autoRickshaw :: TipConfig,
+  taxi :: TipConfig,
+  taxiPlus :: TipConfig
+}
+
+getTipConfig :: String -> TipConfig
+getTipConfig variant = do
+  let city = HU.getCityFromString $ getValueToLocalStore CUSTOMER_LOCATION
+  case city of 
+    Bangalore -> bangaloreConfig variant
+    Kolkata -> defaultTipConfig variant
+    Paris -> defaultTipConfig variant
+    Kochi -> defaultTipConfig variant
+    Delhi -> defaultTipConfig variant
+    Hyderabad -> defaultTipConfig variant
+    Mumbai -> defaultTipConfig variant
+    Chennai -> chennaiConfig variant
+    Coimbatore -> defaultTipConfig variant
+    Pondicherry -> defaultTipConfig variant
+    Goa -> defaultTipConfig variant
+    Pune -> defaultTipConfig variant
+    Mysore -> defaultTipConfig variant
+    Tumakuru -> defaultTipConfig variant
+    AnyCity -> defaultTipConfig variant
+
+mkTipConfig :: Array String -> Array Int -> TipConfig
+mkTipConfig customerTipArray customerTipArrayWithValues = {
+  customerTipArray: customerTipArray,
+  customerTipArrayWithValues: customerTipArrayWithValues
+}
+      
+bangaloreConfig :: String -> TipConfig
+bangaloreConfig variant = 
+  case variant of
+    "SEDAN" -> mkTipConfig [(getString NO_TIP), "₹10 🙂", "₹20 😀", "₹30 😃", "₹40 😁", "₹50 🤩"] [0, 10, 20, 30, 40, 50]
```

Comment:
Can you create function where mkTipConfig take only value and display value is generated out of the input